examples/midifiles/play_midi_file.py

```diff
@@ -18,9 +19,13 @@
 
 with mido.open_output(portname) as output:
     try:
-        for message in MidiFile(filename).play():
+        midifile = MidiFile(filename)
+        t0 = time.time()
+        for message in midifile.play():
             print(message)
             output.send(message)
+        print 'play time: %.2f s (expected %.2f)' % (
+            time.time() - t0, midifile.length)
```

Comment:
Here I would suggest to use a print function compatible with Python 3, possibly also using `.format()`.

Reply:
Thank you!